public/app/plugins/panel/dashlist/module.html

```diff
@@ -9,6 +9,9 @@ <h6 class="dashlist-section-header" ng-show="ctrl.panel.headings">
           <span class="dashlist-title">
             {{dash.title}}
           </span>
+          <span class="dashlist-folder">
+            ({{dash.folderTitle}})
+          </span>
```

Comment:
Maybe we should ignore empty folder names as it looks kind of weird:
![image](https://user-images.githubusercontent.com/562238/90094209-9ae22480-dd2d-11ea-9629-e7312bd3e7ae.png)

Reply:
Hi @hugohaggmark 

I have added a `ng-if` directive to the span. 
Now it does not display the empty folder anymore, when no `folderTitle` property is present.

It should be fine now.

WDYT?